cdap-common/src/test/java/io/cdap/cdap/common/http/AuthenticationChannelHandlerTest.java

```diff
@@ -118,31 +121,35 @@ public void testWriteWithAuditLogging() throws Exception {
   public void testCloseWithAuditLogging() throws Exception {
```

Comment:
Could you please add a test with other order of the calls, so that we test the use ca that was failing?

Reply:
Added for both cases.